tests/functional/disaster-recovery/regional-dr/test_site_failure_recovery_and_failover.py

```diff
@@ -367,27 +416,54 @@ def test_site_failure_and_failover(self, dr_workload, nodes_multicluster):
                         switch_ctx=get_passive_acm_index(),
                     )
                 )
-                time.sleep(5)
+            for wl in discovered_workload:
+                relocate_results.append(
+                    executor.submit(
+                        dr_helpers.relocate(
+                            preferred_cluster=primary_cluster_name,
+                            namespace=wl.workload_namespace,
+                            workload_placement_name=wl.discovered_apps_placement_name,
+                            discovered_apps=True,
+                            old_primary=secondary_cluster_name,
+                            workload_instance=wl,
+                        ),
+                        switch_ctx=get_passive_acm_index(),
+                    )
+                )
+
+                logger.info("Doing Cleanup Operations")
+                dr_helpers.do_discovered_apps_cleanup(
+                    drpc_name=wl.discovered_apps_placement_name,
+                    old_primary=secondary_cluster_name,
+                    workload_namespace=wl.workload_namespace,
+                    workload_dir=wl.workload_dir,
+                    vrg_name=wl.discovered_apps_placement_name,
+                )
```

Comment:
Todo: Will remove it in the next commit as relocate func already has cleanup operations